android/res/layout/fragment_editor.xml

```diff
@@ -5,7 +5,6 @@
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  android:fitsSystemWindows="true"
```

Comment:
It was a part of the bugfix, why did you remove it? https://github.com/organicmaps/organicmaps/pull/3325

Reply:
Ah interesting. For me, it introduced some padding at the top of the editor. (see pr screenshots)

i'll do more testing and find a fix for both cases 👍

Reply:
So, does editor scrollable with open keyboard with your fix?

Reply:
Unfortunately not yet, I'll mark this PR as a draft